src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -28,6 +29,7 @@ const NavHeader = (props) => {
     isFollow
   } = props;
   const navigation = useNavigation();
+  const coverPath = topicDetail?.cover_path || null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of `coverPath` is a good way to handle the optional presence of a cover image. It defaults to `null` if `topicDetail?.cover_path` is undefined, which is a safe approach.
```diff
+ const coverPath = topicDetail?.cover_path || null;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->